### DISCUSSION TITLE
Question regarding new installation

### DISCUSSION THREAD

**Discussion by clear-design** [PrivateBin/PrivateBin#1454] [Q&A]
### Describe the problem/question

Hi there, I'm trying to install a new instance of PrivateBin on veryseriousmessage.com. I've read through the installation instructions, but I get the feeling I'm missing something important. 

After extracting the source in my destination directory, I notice there is no data directory, though I see this referenced everywhere. I've made a data directory with 640 permissions, but I still receive the error "Could not create paste: server error or not responding"'

What am I missing? Are there additional installation instructions somewhere?
Thanks

### Did you use the FAQ section?

- [X] Yes, I have read [the FAQ](https://github.com/PrivateBin/PrivateBin/wiki/FAQ) and I found no solution/answer there.

### What you did?

1. Extracted source code
2. Created conf.php file identical to sample
3. Created 'data' directory, chmod 640
...

### What happens

"Could not create paste: server error or not responding"

### What should happen

Note is successfully stored and link is supplied

### Server address

https://veryseriousmessage.com

### PrivateBin version

1.7.4

### Issue reproducibility

No, I cannot reproduce it on https://privatebin.net.

**Comment by elrido** [comment]
Check your php processes error log to find the reason for it not being able to create the paste.

Likely it is a permission problem: The data folder should get created for you by the software and be owned by the user that the php process is running under.

You may not have created it using the correct user and if you set the flags to 640 you only allow your user to touch to it and no user to execute the folder, which is required to create files within it. The folder needs to have at least the flags 700 (user-read-write-and-executable) and be owned by the user the php process runs as. On Debian and Alpine Linux derivatives that is often the www-data user - check your distros documentation or your service configuration files or use ps to see what it runs under.

**Comment by clear-design** [comment]
Thank for taking the time to respond :) 
I went ahead and started from scratch to see if I could find an error during the data folder creation, but so far no luck.
Unfortunately I'm trying to install this on a sad cPanel instance and don't have full access to ssh or logs. Despite configuring an accessible path for php error logs, I'm not receiving anything in the php logfile or the apache error log. I'll keep at it, but I'm currently at a loss!

**Comment by clear-design** [comment]
Some further digging - same error when trying to configure for MySQL. 
In looking at the console, I'm getting a 401 error from something jQuery is trying to do, which I guess explains why I'm not seeing any PHP errors. Though I can't figure out why I could be getting a 401 error on resources that are obviously loading. 
This is a fresh install, I've changed no permissions.

**Comment by clear-design** [comment]
Alright, so I have found something that works for me. PrivateBin works fine in a subfolder, but not in the domain's root folder. I'm just forwarding to a subfolder via htaccess, and that'll work for my purposes. 

To anyone finding this in the future, let me know if you ever figure it out.

**Comment by officialspitxxx-creator** [comment]
Hola, quiero escribir blogs para mi sitio web https://spitxxx.com También puedo pagar por blogs de buena calidad. Si necesitas ayuda, avísame.
